cli/compose/types/types.go

```diff
@@ -172,7 +172,7 @@ type HealthCheckConfig struct {
 	Timeout     string
 	Interval    string
 	Retries     *uint64
-	StartPeriod string
+	StartPeriod string `mapstructure:"start_period"`
```

Comment:
This field should be a `*time.Duration` not a string. I just noticed that the other fields here are wrong. We'll need to fix that.

Reply:
func convertHealthcheck(healthcheck *composetypes.HealthCheckConfig) (*container.HealthConfig, error) 

will parse and convert the string into time.Duration

Reply:
It will yes, but it's an unnecessary step. The config should be of type `*time.Duration`. This is a mistake that was made in other fields of the HealthCheckConfig (that we need to fix), but there are other fields in the service that use `*time.Duration` correctly.